qiskit_algorithms/optimizers/__init__.py

```diff
@@ -98,6 +98,16 @@
    IMFIL
    SNOBFIT
 
+Qiskit also provides local optimizers based on
+`NLOpt <https://nlopt.readthedocs.io/en/latest/>`_.
+See below section about global optimizers for installation instructions.
```

Comment:
```suggestion
See Global Optimizers section below for the optional NLOpt installation instructions.
```
Changed the wording a little and made it explicit about what its referring to that section for,